frontend/js/require.config.js

```diff
@@ -63,7 +68,7 @@ require.config({
 
         "mediaelementplayer": {
             exports: "mejs",
-            deps: ["jquery"]
+            deps: ["require"]
```

Comment:
Can you explain this to me?

Reply:
Comes from the MEJS documentation: https://github.com/mediaelement/mediaelement/blob/master/docs/usage.md#requirejs